test/test_helper.rb

```diff
@@ -1,5 +1,16 @@
 require 'simplecov'
-SimpleCov.start 'rails'
+require 'simplecov-lcov'
+
+SimpleCov::Formatter::LcovFormatter.config.report_with_single_file = true
+SIMPLECOV_FORMATTERS = [
+  SimpleCov::Formatter::LcovFormatter,
+  SimpleCov::Formatter::HTMLFormatter
+]
+SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new(SIMPLECOV_FORMATTERS)
```

Comment:
Metrics/LineLength: Line is too long. [84/80]

Reply:
Fixed.